src/menu/view/menu.jsx

```diff
@@ -555,14 +577,16 @@ export default class Menu extends Component {
     }
 
     normalizeToArray(items) {
```

Comment:
fix(Menu): passing key which isn't in this.k2n map, causes error

https://github.com/alibaba-fusion/next/issues/975